app/src/main/res/values/arrays.xml

```diff
@@ -7,9 +7,11 @@
         <item>@string/search_type_release</item>
         <item>@string/search_type_label</item>
         <item>@string/search_type_recording</item>
+        <item>@string/search_type_event</item>
         <item>@string/search_type_release_group</item>
         <item>@string/search_type_instrument</item>
-        <item>@string/search_type_event</item>
+        <item>Area</item>
```

Comment:
We do not support searching this types yet, so why these here?

Reply:
Sure, while I was working for the UX part, I added it to see how things would look like. Will remove it for now!